modules/llvm-project/17.0.3.bcr.1/presubmit.yml

```diff
@@ -0,0 +1,48 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  bazel:
+  - "7.x"
```

Comment:
```suggestion
  - "7.x"
  - "8.x"
```

Reply:
I'll add this but I strongly feel Bazel 8 support should not be required for this change if it doesn't immediately work.

Reply:
It never is, but we should always give it a try :⁠-⁠)